bootstrap/components/sources/hdfs-source-topology-component.json

```diff
@@ -33,6 +33,24 @@
         "defaultValue": "/tmp/sourcefiles/",
         "hint": "schema"
       },
+      {
+        "uiName": "Schema Name",
+        "fieldName": "schemaName",
+        "isOptional": false,
+        "tooltip": "Name of the schema to which messages in the given `Source Dir` adhere to.",
+        "type": "string",
+        "defaultValue": "",
+        "hint": "schema"
```

Comment:
@satishd 
You also need to change `type`, `defaultValue` and `hint` here as done in Kafka-source.
https://github.com/hortonworks/streamline/blob/master/bootstrap/components/sources/kafka-source-topology-component.json#L33-L35

Reply:
@shahsank3t Why does it need to be enumstring? Here, schema name should be given by user as a string and it is not populated from other fields like kafka source.

Reply:
@satishd Sorry for the confusion. Yes, we don't need to change this. My bad.